opta/nice_subprocess.py

```diff
@@ -60,65 +74,119 @@ def nice_run(  # type: ignore # nosec
     exit_timeout: Optional[float] = None,
     tee: bool = True,
     check: bool = False,
+    use_new_nice_run=False,
```

Comment:
`old/new` is unclear to me which one I would need to use if I'm writing a new module.

can we come up with a name that carries some meaning about when to use one or the other?
and some comments on this function so someone using this function will understand what are the differences and make a decision about which one to use.

Reply:
Good point, fixe by renaming flag and adding a comment.